<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VisionDialog</class>
 <widget class="QDialog" name="VisionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Vision</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../rc/icons.qrc">
    <normaloff>:/icons/webcam.png</normaloff>:/icons/webcam.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="margin">
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QLabel" name="label_20">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Configurations:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="configs">
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectItems</enum>
         </property>
         <property name="modelColumn">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QPushButton" name="addConfig">
           <property name="text">
            <string>Add</string>
           </property>
           <property name="icon">
            <iconset resource="../rc/icons.qrc">
             <normaloff>:/icons/add.png</normaloff>:/icons/add.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="removeConfig">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Remove</string>
           </property>
           <property name="icon">
            <iconset resource="../rc/icons.qrc">
             <normaloff>:/icons/delete.png</normaloff>:/icons/delete.png</iconset>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="renameConfig">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Rename</string>
           </property>
           <property name="icon">
            <iconset resource="../rc/icons.qrc">
             <normaloff>:/icons/pencil.png</normaloff>:/icons/pencil.png</iconset>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="defaultConfig">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Default</string>
           </property>
           <property name="icon">
            <iconset resource="../rc/icons.qrc">
             <normaloff>:/icons/star.png</normaloff>:/icons/star.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QLabel" name="label_21">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Channels:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="channels">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="modelColumn">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QPushButton" name="addChannel">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
           <property name="icon">
            <iconset resource="../rc/icons.qrc">
             <normaloff>:/icons/add.png</normaloff>:/icons/add.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="removeChannel">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Remove</string>
           </property>
           <property name="icon">
            <iconset resource="../rc/icons.qrc">
             <normaloff>:/icons/delete.png</normaloff>:/icons/delete.png</iconset>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="up">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Up</string>
           </property>
           <property name="icon">
            <iconset resource="../rc/icons.qrc">
             <normaloff>:/icons/arrow_up.png</normaloff>:/icons/arrow_up.png</iconset>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="down">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Down</string>
           </property>
           <property name="icon">
            <iconset resource="../rc/icons.qrc">
             <normaloff>:/icons/arrow_down.png</normaloff>:/icons/arrow_down.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1,0">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Output and Settings:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="CvWidget" name="cv" native="true"/>
       </item>
       <item>
        <widget class="QWidget" name="hsv" native="true">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QHsvPicker" name="visual" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>110</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget" native="true">
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="spacing">
              <number>4</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <item>
                <widget class="QLabel" name="label_16">
                 <property name="minimumSize">
                  <size>
                   <width>80</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Hue</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="_bh"/>
               </item>
               <item>
                <widget class="QLabel" name="label_17">
                 <property name="minimumSize">
                  <size>
                   <width>50</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>to</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="_th"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <item>
                <widget class="QLabel" name="label_18">
                 <property name="minimumSize">
                  <size>
                   <width>80</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Saturation</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="_bs"/>
               </item>
               <item>
                <widget class="QLabel" name="label_19">
                 <property name="minimumSize">
                  <size>
                   <width>50</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>to</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="_ts"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QLabel" name="label_14">
                 <property name="minimumSize">
                  <size>
                   <width>80</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Value</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="_bv"/>
               </item>
               <item>
                <widget class="QLabel" name="label_15">
                 <property name="minimumSize">
                  <size>
                   <width>50</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>to</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="_tv"/>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Source:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="source">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Default Camera</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Parrot AR.Drone Front Camera</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Parrot AR.Drone Bottom Camera</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Save</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CvWidget</class>
   <extends>QWidget</extends>
   <header>cv_widget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QHsvPicker</class>
   <extends>QWidget</extends>
   <header>qhsv_picker.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../rc/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>VisionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>VisionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
